src/Apps/Order/OrderApp.tsx

```diff
@@ -82,13 +83,18 @@ export class OrderApp extends React.Component<OrderAppProps, OrderAppState> {
   render() {
     const { children, location, router, order, params } = this.props
 
+    if (!order) {
+      return <ErrorPage code={404} />
+    }
```

Comment:
@zephraph As we discussed we can't properly return `404` since we don't have access to the `errors`. When the user is not logged in [we can (will) handle that on the Force side](https://github.com/artsy/force/pull/2946), so I'm not extremely concerned about it at this point.

Reply:
If you want to know for sure, you can add an `orderOrError` field.

In fact, I will work on this if nobody else does, because I want a real example for a conf talk I’m working on.

Reply:
👍 Sounds like that can be a follow up item then.